packages/website/src/components/CurriculumSection/TechSliding.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Box } from '@coderscamp/ui/components/Box';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { keyframes } from '@coderscamp/ui/components/Keyframes';
+
+import { techNames } from './TechIcons';
+
+interface TechSlidingProps {
+  direction?: 'normal' | 'reverse';
+}
+
+export const TechSliding = ({ direction = 'normal' }: TechSlidingProps) => {
+  const sliding = keyframes`
+   100% {
+    transform: translateX(-66.6666%);
+  }
+`;
+  const animation = `${sliding} infinite 60s linear ${direction}`;
```

Comment:
💭 We should probably make animation time screen size dependant as for now it looks really slow on mobile devices